scripts/stats/nwps/exevs_nwps_wave_grid2obs_stats.sh

```diff
@@ -0,0 +1,188 @@
+## grid2obs wave model stats 
+#############################
+
+cd $DATA
+echo "Starting grid2obs_stats for ${MODELNAME}_${RUN}"
+
+echo ' '
+echo ' ******************************************'
+echo " *** ${MODELNAME}-${RUN} grid2obs stats ***"
+echo ' ******************************************'
+echo ' '
+echo "Starting at : `date`"
+echo '-------------'
+echo ' '
+
+mkdir -p ${DATA}/gribs
+mkdir -p ${DATA}/ncfiles
+mkdir -p ${DATA}/all_stats
+mkdir -p ${DATA}/jobs
+mkdir -p ${DATA}/logs
+mkdir -p ${DATA}/confs
+mkdir -p ${DATA}/tmp
+mkdir -p ${DATA}/SFCSHP
+
+vhours='00 06 12 18'
+WFO='hgx'
+CG='CG1'
+lead_hours='0 24 48 72 96 120 144'
+
+export GRID2OBS_CONF="${PARMevs}/metplus_config/${STEP}/${COMPONENT}/${RUN}_${VERIF_CASE}"
+
+cd ${DATA}
+
+############################################
+# create point_stat files
+############################################
+echo ' '
+echo 'Creating point_stat files'
+
+for wfo in ${WFO}; do
+	for cg in ${CG}; do 
+		for vhr in ${vhours} ; do
+    			vhr2=$(printf "%02d" "${vhr}")
+    			if [ ${vhr2} = '00' ] ; then
+       				wind_level_str="'{ name=\"WIND\"; level=\"(0,*,*)\"; }'"
+       				htsgw_level_str="'{ name=\"HTSGW\"; level=\"(0,*,*)\"; }'"
+       				perpw_level_str="'{ name=\"PERPW\"; level=\"(0,*,*)\"; }'"
+       				wdir_level_str="'{ name=\"WDIR\"; level=\"(0,*,*)\"; }'"
+    			elif [ ${vhr2} = '06' ] ; then
+       				wind_level_str="'{ name=\"WIND\"; level=\"(2,*,*)\"; }'"
+       				htsgw_level_str="'{ name=\"HTSGW\"; level=\"(2,*,*)\"; }'"
+       				perpw_level_str="'{ name=\"PERPW\"; level=\"(2,*,*)\"; }'"
+       				wdir_level_str="'{ name=\"WDIR\"; level=\"(2,*,*)\"; }'"
+    			elif [ ${vhr2} = '12' ] ; then
+				wind_level_str="'{ name=\"WIND\"; level=\"(4,*,*)\"; }'"
+				htsgw_level_str="'{ name=\"HTSGW\"; level=\"(4,*,*)\"; }'"
+				perpw_level_str="'{ name=\"PERPW\"; level=\"(4,*,*)\"; }'"
+				wdir_level_str="'{ name=\"WDIR\"; level=\"(4,*,*)\"; }'"
+			elif [ ${vhr2} = '18' ] ; then
+       				wind_level_str="'{ name=\"WIND\"; level=\"(6,*,*)\"; }'"
+       				htsgw_level_str="'{ name=\"HTSGW\"; level=\"(6,*,*)\"; }'"
+       				perpw_level_str="'{ name=\"PERPW\"; level=\"(6,*,*)\"; }'"
+       				wdir_level_str="'{ name=\"WDIR\"; level=\"(6,*,*)\"; }'"
+    			fi
+
+    			for lead in ${lead_hours} ; do
+	    			matchtime=$(date --date="${VDATE} ${vhr2} ${lead} hours ago" +"%Y%m%d %H")
+	    			match_date=$(echo ${matchtime} | awk '{print $1}')
+	    			match_hr=$(echo ${matchtime} | awk '{print $2}')
+	    			match_fhr=$(printf "%02d" "${match_hr}")
+	    			flead=$(printf "%03d" "${lead}")
+	    			flead2=$(printf "%02d" "${lead}")
+	    
+	    			EVSINmodelfilename=$COMIN/prep/$COMPONENT/${RUN}.${match_date}/${MODELNAME}/${VERIF_CASE}/${wfo}_${MODELNAME}_${cg}.${match_date}.t${match_fhr}z.f${flead}.grib2
+	    			DATAmodelfilename=$DATA/gribs/${wfo}_${MODELNAME}_${cg}.${match_date}.t${match_fhr}z.f${flead}.grib2
+	    			for OBSNAME in GDAS NDBC; do
```

Comment:
Question, I see GDAS listed as an OBTYPE here but I didn't see anything happening in the prep for the GDAS prepbufr file. Did I miss that?

Reply:
@malloryprow, Right. I have not added GDAS, split_by_subset and related items to ship data because I was not sure if there is ship data available for those nearshore regions. If the ship data available, I can add this OBTYPE later. It would be the topic of the next PR.

Reply:
Okay! Let's take out OBTYPE from this loop for now?

Reply:
@malloryprow, I have just removed the GDAS from stats script.

Reply:
Thank  you!